src/daft-connect/src/connect_service.rs

```diff
@@ -79,17 +79,24 @@ impl SparkConnectService for DaftSparkConnectService {
             }
             OpType::Command(command) => {
                 let command = command.command_type.required("command_type")?;
-
                 match command {
                     CommandType::WriteOperation(op) => {
                         let result = session.execute_write_operation(op, rb).await?;
                         Ok(Response::new(result))
                     }
+                    CommandType::RegisterFunction(_) => todo!(),
```

Comment:
Instead of a `todo!()`, should we convert this into a `not_yet_implemented!(..)`?

Reply:
I think this was a bad merge.